libraries/systemd_install.rb

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
can we add corresponding man page links. for tracking.
like: http://www.freedesktop.org/software/systemd/man/systemd.directives.html